Multisig/Data/Services/SafeTransactionService.swift

```diff
@@ -20,6 +20,10 @@ class SafeTransactionService {
     func safeInfo(at address: String) throws -> SafeStatusRequest.Response {
         return try httpClient.execute(request: SafeStatusRequest(address: address))
     }
+
+    func safeBalances(at address: String) throws -> [SafeBalancesRequest.Response] {
```

Comment:
please add an integration test for that endpoint

Reply:
done